Content(mission): A dead mans path (#7803)
endless-sky/endless-sky: data/human/outfits.txt

	"unplunderable" 1
	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."

outfit "The old mans sword"
	category "Hand to Hand"
	cost 120000
	thumbnail "outfit/old sword"

> 	"capture attack" 7.5
> 	"capture defense" 7.5

Author: ```suggestion
	"capture attack" 3
	"capture defense" 3
```

	"unplunderable" 1

> 	description "A sword that has all kind of alien technology implemented in it. You have yet to find out what it can do. Some functionalities are a personall shield and an implemented shooting mechanism."

Reviewer: wait what? this really is stepping on the toes of my warlord blades xD
What about smth like it cuts trough any materials by heating up? And we imagine this sword is simply Korath in origin?

Reviewer 2: You could tie it in with the warlord blades; their father might have studied a few and built his own, but died before he was able to refine it, or he was found by someone who wanted it and needed to keep it safe.

Reviewer: Well the warlord and Unfettered warlord blades, and although some could have gotten lost I feel like this blade has more of a Korath/Sheragi style, which is good.
Also I intend to give the blades better stats than this since other than the license they're not obtainable and are there to make capturing warlord ships harder gameplay wise. (many story reasons too but anyway)

I think I like the Sheragi idea best though. The Hai, having fought them, could have been inspired. This one could just have been built from the remainders of one of those Sheragi blades (that would have been much bigger).

Reviewer 2: I feel like Sheragi swords would be too big for the player to handle.

Reviewer: Yeah ofc course they would thats why I said this is made out of the remainders of one

> outfit "The old mans sword"
> 	category "Hand to Hand"
> 	cost 120000
> 	thumbnail "outfit/old sword"
> 	"capture attack" 3
> 	"capture defense" 3
> 	"unplunderable" 1
> 	description "A sword that has all kind of alien technology implemented in it. You have yet to find out what it can do. Some functionalities are a personall shield and an implemented shooting mechanism."

Reviewer 2: ```suggestion
outfit "Old Man's Sword"
	category "Hand to Hand"
	cost 120000
	thumbnail "outfit/old sword"
	"capture attack" 3
	"capture defense" 3
	"unplunderable" 1
	description "While swords are seen today as old curios, commemorative objects, or are under archaeological study, this sword seems to be as combat effective as any laser rifle. Swinging the sword discharges an energy bolt that can be used at range to hit an enemy, or neutralize one of their shots coming at you."
```
This is just an idea of what it could look like. Perhaps the description might still be improved.

Reviewer: yeah that could work, so long as its not a personal shield or a gun-sword per say like we intend for the warlord blades it's good for me
I mean even if it is I'm not going to hard block the PR or smth its just that it'd be good to have varied things and if they're the same that'd limit the power of the UHai blade later on

Reviewer 2: Here's a better one:
```suggestion
outfit "Old Man's Sword"
	category "Hand to Hand"
	cost 120000
	thumbnail "outfit/old sword"
	"capture attack" 3
	"capture defense" 3
	"unplunderable" 1
	description "Got this from a pair of siblings from New China, fulfilling their eccentric father's last will and testament. While swords are seen today as old curios, commemorative objects, or are under archaeological study, this sword seems to be as combat effective as any laser rifle, and unlike any you're seen before. Swinging the sword discharges an energy bolt that can be used at range to hit an enemy, or neutralize one of their shots coming at you."
```

Author: I still want to have the part of that you do not really know what this sword can also do in the description. Something like at some occasions the sword showed other functionalities that you were not able to reproduce so far.

> 	description "A sword that has all kind of alien technology implemented in it. You have yet to find out what it can do. Some functionalities are a personall shield and an implemented shooting mechanism."

Reviewer 2: Okay. 
```suggestion
	description "Received this from a pair of siblings from New China, by fulfilling their eccentric father's last will and testament on Prime. 
	While swords are seen today as old curios and commemorative objects as they became obsolete with gunpowder warfare thousands of years ago, this sword seems to be as combat effective as any laser rifle and unlike any you're seen before. Despite how heavy it looks, it is remarkably light, and there are inscriptions on the blade on it that don't appear to be like any human language.
	For whatever reason, the red beam within the blade seems to pulsate as you swing it; perhaps there is more to the sword than you know."
```
After some usage, you might find out about the energy bolts, which replaces the sword with a better one as you get better at using it. The description would add:

Swinging the sword discharges an energy bolt that can be used at range to hit an enemy, or neutralize one of their shots coming at you.

Author: I like this, thanks

	"unplunderable" 1
	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."

> outfit "Old Man's sword"

Reviewer 2: ```suggestion
outfit "Old Man's Sword"
```

	category "Unique"

> 	cost 120000

Reviewer 3: Maybe have a larger base price, since it'll automatically get depreciated to 25% of that value upon receiving it?

	thumbnail "outfit/old sword"
	"capture attack" 3

> 	"capture defense" 3

Reviewer 2: This sword would be one of the best weapons in the game, only Heliarch weapons and the Void Rifle are better. I'd make it go under the Pug Peacekeeping Staff, after all I don't expect a textile worker from New Boston learnt fencing when they were younger. I'm still thinking about that energy bolt thing.

> 	"capture attack" 3
> 	"capture defense" 3

Author: ```suggestion
	"capture attack" 0.95
	"capture defense" 0.95
```

	"unplunderable" 1

> 	description "Received this from a pair of siblings from New China, by fulfilling their eccentric father's last will and testament on Prime."

Reviewer 2: ```suggestion
	description "Received this from a pair of siblings whom you helped out by fulfilling their eccentric father's last will and testament on Prime."
```

> 	description "While swords are seen today as being old curios and commemorative objects, rendered useless when gunpowder became dominant on the battlefield thousands of years ago, this sword is unlike any you're seen before. Despite how heavy it looks, it is remarkably light, and there are inscriptions on the blade on it that don't appear to be like any human language."
> 	description "For whatever reason, the red beam within the blade seems to pulsate as you swing it; perhaps there is more to the sword than you know."

Reviewer 2: ```suggestion
	description "	While swords are seen today as being old curios and commemorative objects, rendered useless when gunpowder became dominant on the battlefield thousands of years ago, this sword is unlike any you're seen before. Despite how heavy it looks, it is remarkably light, and there are inscriptions on the blade on it that don't appear to be like any human language."
	description "	For whatever reason, the red beam within the blade seems to pulsate as you swing it; perhaps there is more to the sword than you know."
```

> outfit "The old mans sword"
> 	category "Hand to Hand"
> 	cost 120000
> 	thumbnail "outfit/old sword"
> 	"capture attack" 3
> 	"capture defense" 3
> 	"unplunderable" 1
> 	description "Received this from a pair of siblings from New China, by fulfilling their eccentric father's last will and testament on Prime. 
> 	While swords are seen today as old curios and commemorative objects as they became obsolete with gunpowder warfare thousands of years ago, this sword seems to be as combat effective as any laser rifle and unlike any you're seen before. Despite how heavy it looks, it is remarkably light, and there are inscriptions on the blade on it that don't appear to be like any human language.
> 	For whatever reason, the red beam within the blade seems to pulsate as you swing it; perhaps there is more to the sword than you know."

Reviewer 2: ```suggestion
outfit "Old Man's sword"
	category "Unique"
	cost 120000
	thumbnail "outfit/old sword"
	"capture attack" 3
	"capture defense" 3
	"unplunderable" 1
	description "Received this from a pair of siblings from New China, by fulfilling their eccentric father's last will and testament on Prime."
	description "While swords are seen today as being old curios and commemorative objects, rendered useless when gunpowder became dominant on the battlefield thousands of years ago, this sword is unlike any you're seen before. Despite how heavy it looks, it is remarkably light, and there are inscriptions on the blade on it that don't appear to be like any human language."
	description "For whatever reason, the red beam within the blade seems to pulsate as you swing it; perhaps there is more to the sword than you know."
```
Sorry, messed it up. I will add the other part about the effectiveness later; my thought was that you level up this sword as you use it.

Author: That would certainly be interesting
